src/commands/delete.js

```diff
@@ -0,0 +1,53 @@
+const utils = require('../utils');
+
+const _delete = (context, appOrVersion, version) => {
+  if (appOrVersion === 'version' && !version) {
```

Comment:
Just realized that the comparison might be better understood if put at the root scope of the call.

E.g. 
```js
const isDeletingVersion = appOrVersion === 'version';
if (isDeletingVersion && !version) {
...
```